Microsoft.Azure.CosmosRepository/src/Microsoft.Azure.CosmosRepository.csproj

```diff
@@ -51,6 +51,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Scrutor" Version="3.3.0" />

```

Comment:
This additional dependency is a bit concerning in that it adds a new dependency for something that is only optional. In other words, even if a user is not using change feeds they'll get this package. Might need to think this over a bit more.

Reply:
I can do this assembly scanning without taking on the package does that suit?